src/pybind/mgr/orchestrator.py

```diff
@@ -650,7 +650,7 @@ def copy_to_this_subinterpreter(r_obj):
         try:
             e = copy_to_this_subinterpreter(c.exception)
         except (KeyError, AttributeError):
-            raise Exception(str(c.exception))
+            raise Exception(type(c.exception).__name__ + ': ' + str(c.exception))
```

Comment:
Maybe cast to a str so that the entire module/class context is captured?

```suggestion
            raise Exception(str(type(c.exception)) + ': ' + str(c.exception))
```